go.mod

```
@@ -2,6 +2,7 @@ module github.com/cloudquery/plugin-sdk/v4
 
 go 1.21.1
 
+replace github.com/cloudquery/plugin-pb-go => /Users/benbernays/Documents/GitHub/plugin-pb-go
```

```suggestion
replace github.com/cloudquery/plugin-pb-go => github.com/cloudquery/plugin-pb-go c8efe8ac52377434221d17800afba8eaba57414c
```
& run `go mod tidy`

It will allow the tests to pass